src/serialization/SerializationService.ts

```diff
@@ -364,29 +390,29 @@ export class SerializationServiceV1 implements SerializationService {
         if (candidate == null) {
             return;
         }
-        this.registerSerializer('!global', candidate);
+        this.registerSerializer(SerializationSymbols.GLOBAL_SYMBOL, candidate);
     }
 
     private static isCustomSerializable(object: any): boolean {
         const prop = 'hzCustomId';
         return (typeof object[prop] === 'number' && object[prop] >= 1);
     }
 
-    private findSerializerByName(name: string, isArray: boolean): Serializer {
-        let convertedName: string;
-        if (name === 'number') {
-            convertedName = this.serializationConfig.defaultNumberType;
-        } else if (name === 'buffer') {
-            convertedName = 'byteArray';
-        } else {
-            convertedName = name;
-        }
-        const serializerName = convertedName + (isArray ? 'Array' : '');
-        const serializerId = this.serializerNameToId[serializerName];
-        if (serializerId == null) {
-            return null;
+    // eslint-disable-next-line @typescript-eslint/ban-types    
+    private findSerializerByName(clazz: Function | Symbol, isArray: boolean): Serializer {
```

Comment:
```suggestion
    private findSerializerForType(clazz: Function | Symbol, isArray: boolean): Serializer {
```

or findSerializerByType

Reply:
Or I think it will be cleaner if we split this function into two: 

`findSerializerForClass` and `findSerializerForSymbol`

Reply:
I prefer "findSerializerByType".